src/app/layout.tsx

```diff
@@ -19,15 +21,58 @@ async function fetchDaoMetrics() {
   return fetchMetrics();
 }
 
+const standard = {
+  primary: "#171717",
+  secondary: "#404040",
+  tertiary: "#737373",
+  neutral: "#FFFFFF",
+  wash: "#FAFAFA",
+  line: "#EAEAEA",
+  positive: "#00992B",
+  negative: "#C52F00",
+  brandPrimary: "#171717",
+  brandSecondary: "#FFFFFF",
+};
+
+const defaults = standard;
+
 export default async function RootLayout({
   children,
 }: {
   children: React.ReactNode;
 }) {
   const metrics = await fetchDaoMetrics();
+  const { ui } = Tenant.current();
+
+  const primary = ui?.customization?.primary || defaults.primary;
+  const secondary = ui?.customization?.secondary || defaults.secondary;
+  const tertiary = ui?.customization?.tertiary || defaults.tertiary;
+  const neutral = ui?.customization?.neutral || defaults.neutral;
+  const wash = ui?.customization?.wash || defaults.wash;
+  const line = ui?.customization?.line || defaults.line;
+  const positive = ui?.customization?.positive || defaults.positive;
+  const negative = ui?.customization?.negative || defaults.negative;
+  const brandPrimary = ui?.customization?.brandPrimary || defaults.brandPrimary;
+  const brandSecondary =
+    ui?.customization?.brandSecondary || defaults.brandSecondary;
+  const font = ui?.customization?.font || inter.style.fontFamily;
+
+  const style = {
+    "--primary": primary,
+    "--secondary": secondary,
+    "--tertiary": tertiary,
+    "--neutral": neutral,
+    "--wash": wash,
+    "--line": line,
+    "--positive": positive,
+    "--negative": negative,
+    "--brand-primary": brandPrimary,
+    "--brand-secondary": brandSecondary,
+    "font-family": font,
+  } as React.CSSProperties;
```

Comment:
@andreitr here is the other place I'd like to get your feedback on -- this is the other half of how we pull the theme from the tenant. 

If you agree with these two files, the rest of the code is just ripping out css and updating them to use these theme variables. I might not have caught 100% of them but I think I did a decent job and we can always catch the rest as we see them later.

Goal is for no glaringly obvious style degradations

Reply:
I think this makes sense. My initial reaction was to return `React.CSSProperties` from the tenant but I see why you chose to assign these values in the layout. LGTM